### PR TITLE
add middleware

### PR DESCRIPTION
**Pull request by AlecErasmus**

### CODE REVIEW

File: landing/src/app/(auth)/sign-in/page.tsx [hunk 1]
```diff
@@ -1,41 +1,28 @@
...
-            // router.push(`/home`)
-        }
-    };
-
+    // const disableLogin = !ready || (ready && authenticated);
 
     useEffect(() => {
-        if (authenticated && user != null) {
-            handleUserLogin();
+        console.log(authenticated);
```
Comment (sudoFerraz) [inline landing/src/app/(auth)/sign-in/page.tsx:19]: console.log

File: landing/src/app/(auth)/sign-in/page.tsx [hunk 2]
```diff
@@ -1,41 +1,28 @@
...
+        if (authenticated) {
+            // for some reason the styles are not being loaded if you use outer.push
```
Comment (sudoFerraz) [inline landing/src/app/(auth)/sign-in/page.tsx:21]: is this privy?

File: landing/src/app/(auth)/sign-out/page.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <BackgroundGradient className="rounded-[22px] max-w-sm p-4 sm:p-10 bg-zinc-900">
+                <Image
+                    width={400}
+                    height={400}
+                    alt="OnlyFriends Text Logo"
+                    src={"https://xkrrcuvbmddexwwj.public.blob.vercel-storage.com/logo-with-text-sOnqDTsKFHSkmsrGeenkodbxLtlqR2.svg"}
+                />
+            </BackgroundGradient>
+        </div>
+    )
```
Comment (sudoFerraz) [inline landing/src/app/(auth)/sign-out/page.tsx:48]: on the future we should use the standard of web3 apps, that don't have a signin or signout page, its all on the header, and if its not connect then they just don't display the data

File: landing/src/app/layout.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { GeistSans } from "geist/font/sans";
-import './globals.css'
 import { cn } from '@/lib/utils';
+import './globals.css'
```
Comment (sudoFerraz) [inline landing/src/app/layout.tsx:3]: oh, okay, this might be it, don't import globals.css on other layouts, only on the root one.

**Review by sudoFerraz — APPROVED** [review]

File: landing/src/app/(auth)/sign-in/page.tsx (hunk 2 above)
Comment (AlecErasmus) [reply landing/src/app/(auth)/sign-in/page.tsx:21]: no its next js

File: landing/src/app/(auth)/sign-out/page.tsx (hunk 3 above)
Comment (AlecErasmus) [reply landing/src/app/(auth)/sign-out/page.tsx:48]: Noted, we can think how todo that next

**Pull request merged**